src/components/pages/AddNewBookForm.js

```diff
@@ -2,7 +2,36 @@ import React from 'react';
 
 function AddNewBookForm() {
   return (
-    <div>AddNewBookForm</div>
+    <div className="main-container">
+      <p>Add a New Book </p>
+      <form action="" className="add-book-form">
+
+        <label htmlFor="title">
+          Title
+          <input
+            type="text"
+            placeholder="Add a title"
+            id="title"
+          />
+        </label>
+
+        <label htmlFor="author">
+          Author
+          <input
+            type="text"
+            placeholder="Add the book author"
+            id="author"
+          />
+        </label>
+
+        <button
+          type="button"
+        >
+          Submit
+        </button>
+
+      </form>
+    </div>
   );
 }
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As required for the project,I think it would be better if you create add an appropriate `Route` to the `CategoriesPage` in the `App.js` file as it misses there.
```jsx
function App() {
  return (
    <BrowserRouter>
      <Navbar />
      <Routes>
        <Route path="/" element={<Books />} />
        <Route path="/categories" element={<CategoriesPage />} />
        ....
      </Routes>
    </BrowserRouter>
  )
}
```

Reply:
Thank you @tresorsawasawa for the comment. I think this may be a question of commit history. Here is the updated code of my App.js.
https://github.com/diarisdiakite/bookstore/pull/1